### PR TITLE
Plugin manager - fixed config.go for UT's , rearranged files for common build 

### PR DESCRIPTION
**Pull request by GouthamBanala**
config.go fix to make engine UT's work
Now moved all plugin manager files under lom to make it work with common build. 
Fixed all the indentations

### CODE REVIEW

File: lom/src/lib/lomcommon/helper.go [hunk 1]
```diff
@@ -225,12 +225,12 @@ func GetlogPeriodic() *logPeriodic_t{
     }    
     // TODO: Replace with global abort channel later
     chAbort := make(chan interface{})
-    logPeriodicInit(chAbort)
+    LogPeriodicInit(chAbort)
```
Comment (renukamanavalan) [inline lom/src/lib/lomcommon/helper.go]: Why are we exporting ?

**Review by renukamanavalan — COMMENTED** [review]
no more tabs?

File: lom/src/lib/lomcommon/helper.go (hunk 1 above)
Comment (GouthamBanala) [reply lom/src/lib/lomcommon/helper.go]: reverted it now

**Review by renukamanavalan — APPROVED** [review]
Based on Goutham's comments "No logical/functional" code changes.

**Review by prithviraghavtm — APPROVED** [review]

**Pull request merged**